exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/ServiceWrapper.java

```diff
@@ -41,6 +46,26 @@
  */
 final class ServiceWrapper {
 
+  /**
+   * Default interface comprised of transactions defined in the service implementation
+   * (intrinsic to this service).
+   * todo: @slowli What's the proper term for this interface (intrinsic, implicit, default)?
```

Comment:
@slowli Is there one?

Reply:
Pink @slowli

Reply:
Sorry, didn't notice the ping. The core [uses "default"](https://github.com/exonum/exonum/blob/38f5d3d73fa372b6803a54fb16d640eb75e59f18/exonum/src/runtime/mod.rs#L361), however, I'd not consider this a 100% settled term. Do you think another term from the list is better? IMO, there's a tie between "default" and "intrinsic".

Reply:
TIL default is not an adjective. 

I agree that both make sense: [intrinsic](https://www.oxfordlearnersdictionaries.com/definition/english/intrinsic?q=intrinsic) says that this interface belongs to, or is a part of a service artifact (with no extra definition elsewhere); [default](https://www.lexico.com/en/definition/default) says that that interface is used when no interface is specified.

Reply:
["implicit"](https://www.oxfordlearnersdictionaries.com/definition/english/implicit?q=implicit) will probably highlight that there is no explicit definition, but I don't think that is what must be communicated.